website/content/docs/connect/cluster-peering/usage/create-sameness-groups.mdx

```diff
@@ -203,18 +198,20 @@ For more information about exporting services, including examples of configurati
 After you create each exported services configuration entry, apply it to the Consul server with the following CLI command:
 
 ```shell-session
-$ consul config write group-a-export.hcl
+$ consul config write product-sg-export.hcl
 ```
 
-#### Export services for cluster peers and sameness groups at the same time
+#### Export services for cluster peers and sameness groups as part of the same workflow
+
+If you separately follow the steps in this documentation to create a cluster peering connection between two partitions and then add those partitions to a sameness group, you write and apply two separate exported services configuration entries: one to export services to the peer, and a second to export services to other members of the group.
 
-It is possible to combine the exported services configuration entry for creating sameness groups with the entry required to [export services when establishing cluster peering connections](/consul/docs/connect/cluster-peering/usage/establish-cluster-peering#export-services-between-clusters).
+If you are peering clusters with the goal of creating a sameness group, it is possible to combine these workflows by using a single exported services configuration entry.
 
-In this workflow, configure the `Services[].Consumers` block with the `SamenessGroup` field instead of the `Peer` field. You should write the `sameness-group` configuration entry to Consul before referencing it in the `exported-services` configuration entry.
+Configure the `Services[].Consumers` block with the `SamenessGroup` field instead of the `Peer` field. Be aware that this scenario requires you to write the `sameness-group` configuration entry to Consul before you apply the `exported-services` configuration entry that references the sameness group.
```

Comment:
I'm not even sure if this is a true statement. If an exported services entry is created and references a sameness group that doesn't exist, then it will be a no-op.